.github/workflows/docker-publish.yml

```diff
@@ -52,6 +52,15 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
 
+      - name: Set output
+        id: vars
+        run: |
+          TAG=${echo $(echo $REF | cut -d "/" -f3):-latest}
```

Comment:
```
    ${parameter:-word}    Use Default Values.  If parameter is unset or null,
                          the expansion of word is substituted; otherwise,
                          the value of parameter is substituted.
```

I think you are trying to use a value here and not a variable; the substitutor want to substitute the content of a variable.

Reply:
Is it something along the lines of
```sh
if [ "$(echo $REF | cut -d "/" -f3)" ]; then
        TAG=$REF
else
        TAG=latest
fi
```
you want?

Reply:
ahhh maybe.

Once I get the action fixed then it should be good to go.

Docker hub deploys seem to be working